Add troubleshooting section to Husqvarna Automower (#33436)
home-assistant/home-assistant.github.io: source/_integrations/husqvarna_automower.markdown

{% include integrations/config_flow.md %}

## Troubleshooting

> If you have an error with your credentials, you can delete them in the [application credentials](/integrations/application_credentials/) user interface.

Reviewer: ```suggestion
If you have an error with your credentials, you can delete them on the [application credentials](/integrations/application_credentials/) page.
```
